src/parser.js

```diff
@@ -47,23 +46,102 @@ function checkWarnings(warnings){
   }
 }
 
+// function to clean up links 
+
+
+// creating function to trak length of removing items 
+
+
+
+function checker(begining, ending){
+  let arguments = {
+    start: begining+1,
+    end: begining+ (ending-(begining+1)+2)
```

Comment:
looks interesting but strange. i'm fine to add more lines and make it less confusing